src/components/layout/AppContent/AppContent.tsx

```diff
@@ -84,7 +84,7 @@ export const AppContent = () => {
                 key={`${key}-slide-big-container`}
             >
                 <div
-                    className="content-container__movie-series-content"
+                    className={`content-container__movie-series-content ${index === 4 || index === 5 ? "start-row-center-flex" : ""}`}
```

Comment:
Maybe we can use approach from this article https://habr.com/ru/articles/649381/